traitsui/image/image.py

```diff
@@ -5,6 +5,6 @@
 warn(
     DeprecationWarning(
         "traitsui.image has been moved to pyface.image and will be removed in "
-        "TraitsUI version 6.0"
+        "TraitsUI version 7.2"
```

Comment:
Should we still mention it was originally intended to be removed in 6.0 and now will be removed in 7.2?  This implies it has been on its way out for quite some time.  It may not matter, but I thought I remembered that from the stand up.

Reply:
Sure!